src/async-components/views/dialogs/eventindex/ManageEventIndexDialog.tsx

```diff
@@ -99,17 +100,20 @@ export default class ManageEventIndexDialog extends React.Component<IProps, ISta
         let crawlingRoomsCount = 0;
         let roomCount = 0;
         let eventCount = 0;
-        let currentRoom = null;
+        let currentRoom: string | null = null;
 
         const eventIndex = EventIndexPeg.get();
+        if (!eventIndex) return;
```

Comment:
The if-condition on line 108 can then be removed.

Reply:
Removed L106 instead, not sure why i even added it tbh